.yamato/ruamel/config/hdrp_hybrid.metafile

```diff
@@ -4,7 +4,7 @@ project:
 test_platforms:
   - playmode
   - playmode_XR
-  - editmode
+  # - editmode
```

Comment:
is this the only metafile change needed?

Reply:
yes should be, this will just exclude editmode ymls from being created